src/test/java/org/jabref/logic/util/io/FileUtilTest.java

```diff
@@ -477,4 +478,28 @@ void legalPaths(String fileName) {
     void illegalPaths(String fileName) {
         assertTrue(FileUtil.detectBadFileName(fileName));
     }
+
+    @ParameterizedTest
+    @CsvSource({
+            "''                             ,                                     ,   ",
+            "''                             ,                                     , -3",
+            "''                             ,                                     ,  0",
+            "''                             ,                                     ,  3",
+            "''                             ,                                     ,  5",
+            "''                             ,                                     , 10",
+            "''                             , thisisatestfile.pdf                 ,   ",
+            "''                             , thisisatestfile.pdf                 , -5",
+            "''                             , thisisatestfile.pdf                 ,  0",
+            "...                            , thisisatestfile.pdf                 ,  3",
+            "...                            , thisisatestfile.pdf                 ,  5",
```

Comment:
Please! Use all 5 characters: `th...`.

Reply:
Definitely, I will modify the functionality to make use of the available characters.